tardis/model.py

```diff
@@ -250,116 +250,44 @@ def save_spectra(self, fname):
         self.spectrum.to_ascii(fname)
         self.spectrum_virtual.to_ascii('virtual_' + fname)
 
-
-    def to_hdf5(self, buffer_or_fname, path='', close_h5=True):
+    def to_hdf(self, path_or_buf, path='', close_hdf=True,
+               plasma_properties=None):
         """
-            This allows the model to be written to an HDF5 file for later analysis. Currently, the saved properties
-            are specified hard coded in include_from_model_in_hdf5. This is a dict where the key corresponds to the
-            name of the property and the value describes the type. If the value is None the property can be dumped
-            to hdf via its attribute to_hdf or by converting it to a pd.DataFrame. For more complex properties
-            which can not simply be dumped to an hdf file the dict can contain a function which is called with
-            the parameters key, path, and  hdf_store. This function then should dump the data to the given
-            hdf_store object. To dump  properties of sub-properties of  the model, you can use a dict as value.
-            This dict is then treated in the same way as described above.
+        Store the model and the runner to an HDF structure.
 
         Parameters
         ----------
+        path_or_buf
+            Path or buffer to the HDF store
+        path : str
+            Path inside the HDF store to store the plasma
+        close_hdf : bool
```

Comment:
No, if we go with passing the `path_or_buf` straight to pandas, we don't. I removed it in another fixup for model. It will need to be removed from the rest `to_hdf` methods too.